micromanager_gui/_gui_objects/_sample_explorer_widget.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, parent=None):
         self.move_to_Button.clicked.connect(self.move_to)
         self.browse_save_explorer_Button.clicked.connect(self.set_explorer_dir)
 
-        self.stop_scan_Button.clicked.connect(lambda e: self._mmc.cancel())
+        self.stop_scan_Button.clicked.connect(lambda e: self._mmc._mda_engine.cancel())
```

Comment:
This should use the public API in case we change the pymmcore-plus internals later.

```suggestion
        self.stop_scan_Button.clicked.connect(lambda e: self._mmc.mda.cancel())
```